### ISSUE TITLE
False Positive on TestPlatform nuget package (vstest)

### ISSUE THREAD

**Issue by tobyash86** [dependency-check/DependencyCheck#3000]
I am looking for confirmation if findings on TestPlatform NuGet package are false positives or true positives.

Attachments:
- The location of the dependency: https://www.nuget.org/packages/Microsoft.TestPlatform/
- The CPE that is believed to be false positive: please review my investigation on vstest project issue - https://github.com/microsoft/vstest/issues/2620

#### Example
False positive on library Microsoft.VisualStudio.TestTools.UITest.Extension.IE64.Communication.dll - reported as cpe:2.3:a:microsoft:visual_studio:*:*:*:*:*:*:*:*

**Comment by jeremylong** [comment]
Unable to replicate.  I ran the following 9where dotnet_fp is a sample project I use for testing)

```
cd ~/Projects/samples/dotnet_fp
dotnet add package Microsoft.TestPlatform --version 16.8.3
dependency-check.sh -o .--enableExperimental -s ~/Projects/samples/dotnet_fp
```

These were not identified.  How are you scanning?

**Comment by tobyash86** [comment]
Ok, here are my steps:

1. Download latest OWASP DependencyCheck release
2. Download [TestPlatform](https://www.nuget.org/packages/Microsoft.TestPlatform/) v.16.8.3 NuGet package
3. Extract contents of TestPlatform package into separate folder
4. Run DependencyCheck scan on that folder

My script:

`./dependency-check/bin/dependency-check.bat -o ./out -s ./vstest`

where `vstest` denotes folder where `Microsoft.TestPlatform` package has been extracted

**Comment by tobyash86** [comment]
Hi, 
Is there any progress on this? Do you need any additional assistance?

**Comment by jeremylong** [comment]
suppression rules have been added and will be included in 6.0.5.
